lib/mastodon/emoji_cli.rb

```diff
@@ -96,5 +145,22 @@ def color(green, _yellow, red)
         :red
       end
     end
+
+    def export_file_name(emoji)
+      if options[:shortcode]
+        emoji.shortcode + '.png'
```

Comment:
Could use `emoji.shortcode + File.extname(emoji.image_file_name)` to be future-proof for file formats.

Reply:
Awesome, thanks 😊